src/initializers/newrelic/index.ts

```diff
@@ -3,8 +3,8 @@ import requireInjected from '../../require-injected';
 
 let newrelic;
 export default async (config, orkaOptions: Partial<OrkaOptions>) => {
+  process.env.NEW_RELIC_HOME = __dirname;
```

Comment:
Is this to be used for development environments?

Reply:
No.

Reply:
👍